README.md

```diff
@@ -610,4 +610,4 @@ Private blockchains me support the ability for DID users to control exclusion of
 | --- | --- | --- | --- |
 | DID Resolver | Javascript | [npm package](https://www.npmjs.com/package/eosio-did-resolver) | [https://github.com/Gimly-Blockchain/eosio-did-resolver](https://github.com/Gimly-Blockchain/eosio-did-resolver) |
 | DID Operations (CRUD) | Javascript |  [npm package](https://www.npmjs.com/package/eosio-did) | [https://github.com/Gimly-Blockchain/eosio-did](https://github.com/Gimly-Blockchain/eosio-did) |
-| Universal Resolver Driver | Docker | Docker Hub | [https://github.com/Gimly-Blockchain/eosio-did-driver](https://github.com/Gimly-Blockchain/eosio-did-driver) |
\ No newline at end of file
+| Universal Resolver Driver | Docker | Docker Hub | [https://github.com/Gimly-Blockchain/eosio-did-driver](https://github.com/Gimly-Blockchain/eosio-did-driver) |%
```

Comment:
is this right?

Reply:
ups I guess I introduced my own typo, fixed now.